ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import React from 'react'
+import styled from '../../styles/styled-components'
+import withListenTo from '../../container/backbone-container'
+import MarionetteRegionContainer from '../../container/marionette-region-container'
+const Backbone = require('backbone')
+const CustomElements = require('../../../js/CustomElements.js')
+const NotificationGroupView = require('../../../component/notification-group/notification-group.view.js')
+const user = require('../../../component/singletons/user-instance.js')
+const moment = require('moment')
+const userNotifications = require('../../../component/singletons/user-notifications.js')
+
+const Empty = styled.div`
+  transition: transform ${props => props.theme.coreTransitionTime} linear;
+  transform: scale(1);
+  text-align: center;
+  font-size: ${props => props.theme.largeFontSize}
+  padding: ${props => props.theme.mediumSpacing}
+`
+const Notifications = styled.div`
+  height: 100%;
+  width: 100%;
+  display: block;
+  overflow: auto;
+  padding: ${props => props.theme.mediumSpacing};
+`
+
+class UserNotifications extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = userNotifications
```

Comment:
I don't think we really want a Backbone collection as our state here. We're not using this value anywhere either, just using it to re-render.
There's 2 options here - set the state to `userNotifications.toJSON()` and then do the same in the setState call below and use `this.state.userNotifications.length === 0` on line 60
or
- remove this from the state, it's not needed. and have the `listenTo` callback just be `this.setState({})` to trigger the re-render